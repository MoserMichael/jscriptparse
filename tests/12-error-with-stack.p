    def rec(num) {
        if num == 0 {
            return 1 / 0
        }
        return rec(num - 1)
    }        
    
    rec(5)
