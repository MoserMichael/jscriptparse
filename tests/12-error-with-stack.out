Error: Can't divide by zero
#(tests/12-error-with-stack.p:5)             return 1 / 0
                               |......................^
#(tests/12-error-with-stack.p:3)     def rec(num) {
                               |.....^
#(tests/12-error-with-stack.p:7)         return rec(num - 1)
                               |................^
#(tests/12-error-with-stack.p:3)     def rec(num) {
                               |.....^
#(tests/12-error-with-stack.p:7)         return rec(num - 1)
                               |................^
#(tests/12-error-with-stack.p:3)     def rec(num) {
                               |.....^
#(tests/12-error-with-stack.p:7)         return rec(num - 1)
                               |................^
#(tests/12-error-with-stack.p:3)     def rec(num) {
                               |.....^
#(tests/12-error-with-stack.p:7)         return rec(num - 1)
                               |................^
#(tests/12-error-with-stack.p:3)     def rec(num) {
                               |.....^
#(tests/12-error-with-stack.p:7)         return rec(num - 1)
                               |................^
#(tests/12-error-with-stack.p:3)     def rec(num) {
                               |.....^
#(tests/12-error-with-stack.p:10)     rec(5)
                                |.....^
